challenges/2021_CyrilMorcrette_cloudBaseHeight/cbh_challenge_send_data_to_zarr.ipynb

```diff
@@ -0,0 +1,2569 @@
+{
```

Comment:
<p>I'd say rather than commenting out the code, you might rather but the code in an if block. My preference is not to have any commented out code in the final notebook if possible, i think that can be confusing as to when it should be used or not. Sometimes it is required and useful, but often I think an if/else block can be helpful in showing whats happening. For example here it could be:</p><pre class="ql-syntax" spellcheck="false">do_positional_encoding = False
if do_positional_encoding:
    #positional encoding code goes here
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>